conduct.md

```diff
@@ -189,17 +209,17 @@ guidelines. While the Honor Code involves all students in Allegheny College and
 is enforced by the college's Honor Committee, this code of conduct only
 includes members of the Software Engineering class for the spring semester of 2021..
 
-*Infractions shall be imposed at three different levels of severity:*
+_Infractions shall be imposed at three different levels of severity:_
```

Comment:
Can you please explain the purpose of this change. It could potentially introduce unnecessary inconsistency with other branches.

Reply:
Please don't forget to address this request